spark-operator.advisories.yaml

```
@@ -0,0 +1,69 @@
+schema-version: "2"
+
+package:
+  name: spark-operator
+
+advisories:
+  - id: CVE-2023-2431
+    events:
+      - timestamp: 2023-09-07T13:45:23Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-in-execution-path
+          note: Vulnerable code is part of the Kubernetes kubelet which is not in the execution path of the spark-operator
+
+  - id: CVE-2023-2727
+    events:
+      - timestamp: 2023-08-11T19:41:17Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-in-execution-path
+          note: Vulnerable code is part of the Kubernetes API server which is not in the execution path of spark-operator
```

It looks like this code can't be imported, so let's say not included in package.

```suggestion
          type: vulnerable-code-not-included-in-package
          note: Vulnerable code is part of the Kubernetes API server and not importable.
```

---

thank you!